src/decoder.rs

```diff
@@ -595,7 +603,7 @@ impl<'a> Scope<'a> {
         }
     }
 
-    pub fn iter(&self) -> impl Iterator<Item = (Cow<'static, str>, Value)> {
+    pub fn iter(&'a self) -> impl Iterator<Item = (&'a Cow<'static, str>, &'a Value)> {
```

Comment:
I believe that lifetlime elision can be used to avoid the need for `'a` here:

```suggestion
    pub fn iter(&self) -> impl Iterator<Item = (&Cow<'static, str>, &Value)> {
```

But I would have to double-check that this would work...

Reply:
Thanks, fixed.